src/Pages/License/Generate/GenerateLicense.jsx

```diff
@@ -4,11 +4,14 @@ import { styled } from "@mui/styles";
 import CardContent from "@mui/material/CardContent";
 import Stack from "@mui/material/Stack";
 import Button from "@mui/material/Button";
-import { useState } from "react";
+import { useState, useContext } from "react";
+import { useNavigate } from "react-router-dom";
+import { AppContext } from "App";
 import CustomerDetails from "./CustomerDetails";
 import ProductDetails from "./ProductDetails";
 import FeatureDetails from "./FeatureDetails";
 import LimitationDetails from "./LimitationDetails";
+import OptionalDetails from "./OptionalDetails";
 
 export const CustomCard = styled(Card)(() => ({
```

Comment:
If these are used by some other component also, move them to a utility file. This does not look related to Generate License